lib/bundler/cli/remove.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Bundler
+  class CLI::Remove
+    def initialize(gems)
+      @gems = gems
+      @removed_deps = []
+    end
+
+    def run
+      builder = Dsl.new
+      builder.eval_gemfile(Bundler.default_gemfile)
+
+      # store removed gems to display on successfull removal
+      @removed_deps = builder.remove_gems(@gems)
+
+      # remove gems from lockfile
+      @definition = builder.to_definition(Bundler.default_lockfile, {})
```

Comment:
looks like this doesnt need to be an instance variable

Reply:
Removing it.